cmd/serve_cmd.go

```diff
@@ -32,6 +37,12 @@ func serve(ctx context.Context) {
 	}
 	defer db.Close()
 
+	// Run separate server for profiler
+	if config.Profiler.Enabled {
+		go func() {
+			logrus.Warning("error running profiler: ", http.ListenAndServe(config.Profiler.Addr, nil))
+		}()
+	}
```

Comment:
I think this needs proper shutdown support? In any case can you move it to the `observability` package?